pkg/uplinkdb/uplinkdb_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplinkdb_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/uplinkdb"
+	"storj.io/storj/satellite"
+	"storj.io/storj/satellite/satellitedb/satellitedbtest"
+)
+
+func TestUplinkdb(t *testing.T) {
+	satellitedbtest.Run(t, func(t *testing.T, db satellite.DB) {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
+
+		testDatabase(ctx, t, db.UplinkDB())
+	})
+}
+
+func testDatabase(ctx context.Context, t *testing.T, upldb uplinkdb.DB) {
```

Comment:
No need to have separate methods, move it to TestUplinkdb

Reply:
Felt it was easy to follow code as it is now.I moved that function in to TestUplinkdb as suggested, and  aesthetically didn't look appealing.